packages/docs/src/components/page-configs/ui-elements/va-card/page-config.ts

```diff
@@ -1,120 +1,127 @@
-import { BlockType, ApiDocsBlock } from '../../../../types/configTypes'
-import VaCard
-  from 'vuestic-ui/src/components/vuestic-components/va-card/VaCard.vue'
-import apiOptions from './api-options'
+import { BlockType, ApiDocsBlock } from "../../../../types/configTypes";
+import VaCard from "vuestic-ui/src/components/vuestic-components/va-card/VaCard.vue";
+import apiOptions from "./api-options";
 
 export default [
   {
     type: BlockType.TITLE,
-    translationString: 'card.title',
+    translationString: "card.title"
```

Comment:
please one tick :X.

Reply:
can i added the prettier config to the project?

Reply:
returned the previous formatting

Reply:
That's just ide config, no?